validate.py

```diff
@@ -196,5 +209,21 @@ def save_views_imgs(lines, size=(500, 500), interactive=False, ext='jpg'):
     results = add_header_properties(results, header)
     print(results)
 
+    lengths = np.array(list(length(streamlines)), dtype=np.float).tolist()
```

Comment:
Also, the "x" (length) array for histrogram contains too many values. Is it storing values for *every* fibers? product.json is too big .. because of it. I should be a few KB.

 Another thing.. maybe we can save out the histogram data under `secondary` directory - maybe as a CSV? Any data stored under secondary can be used to do group analysis later - across all subjects.